scss/_patterns_subnav.scss

```diff
@@ -0,0 +1,55 @@
+@import 'settings';
+
+@mixin vf-p-subnav {
+  .p-subnav {
+    @extend %vf-is-bordered;
+    @extend %vf-has-box-shadow;
+    @extend %vf-has-round-corners;
+    margin: 0;
+    padding: 0;
+    z-index: 5;
+
+    @media (min-width: $breakpoint-navigation-threshold + 1) {
+      clip: rect(0, 1000px, 1000px, -10px); /* Clip the top of the box-shadow off */
+      margin-top: -1px;
+      position: absolute;
+    }
+
+    @media (max-width: $breakpoint-navigation-threshold) {
+      border: 0;
+      box-shadow: none;
+    }
+  }
+
+  .p-subnav__toggle > a {
+    padding-right: $sph-inner--x-large;
+  }
+
+  .p-subnav__toggle-indicator {
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='4' width='10'%3E%3Cpath d='M3.637 3.138A26.335 26.335 0 0 1 0 0h1.541a21.242 21.242 0 0 0 1.364 1.187 16.899 16.899 0 0 0 .752.563c.397.282.935.626 1.315.848h.054c.38-.222.918-.566 1.315-.848.398-.28.788-.583 1.169-.904.327-.275.643-.557.947-.846h1.541a26.335 26.335 0 0 1-3.637 3.138c-.519.365-.973.652-1.362.862-.39-.21-.844-.497-1.362-.862z' fill='%23666' fill-rule='evenodd'/%3E%3C/svg%3E");
```

Comment:
Can this include the icon to save on size.

Reply:
@anthonydillon The chevron icon has been removed so I'll need to add the SVG here if we want to use it.